geomstats/geometry/poincare_polydisk.py

```diff
@@ -105,16 +105,15 @@ class PoincarePolydiskMetric(ProductRiemannianMetric):
     """
 
     default_coords_type = 'extrinsic'
-    default_point_type = 'vector'
 
     def __init__(self, n_disks, coords_type='extrinsic'):
         self.n_disks = n_disks
         self.coords_type = coords_type
-        self.default_point_type = 'matrix'
         list_metrics = []
         for i_disk in range(n_disks):
             scale_i = (n_disks - i_disk) ** 0.5
             metric_i = HyperboloidMetric(2, coords_type, scale_i)
             list_metrics.append(metric_i)
         super(PoincarePolydiskMetric, self).__init__(
             metrics=list_metrics, default_point_type='matrix')
+        self.default_point_type = 'matrix'
```

Comment:
This might actually not be needed since the line with super() should attribute 'matrix' to self.default_point_type already?

Reply:
Probably, I might have used this for debugging. I'll check